genie-docs/src/docs/asciidoc/_properties.adoc

```diff
@@ -368,6 +368,11 @@ of 2.0
 |5
 |no
 
+|genie.scripts-manager.refresh-interval
```

Comment:
is there any desire to have this on a per script basis instead of globally refresh all scripts at the same interval

Reply:
As discussed in person: per-script refresh would high-effort, low-reward, so leaving it out, until we have a strong use-case for it.